src/remote.sh

```diff
@@ -1,5 +1,6 @@
 # NOTE: It is recommended that src/kw_config_loader.sh be included before this
 # file
+include $KW_LIB_DIR/kw_config_loader.sh
```

Comment:
I did not understand why this change deserves a single commit. See if it makes sense to squash this commit with the previous one; if not, please elaborate a little bit more on the commit message.